.github/workflows/sync-wiki.yml

```diff
@@ -0,0 +1,30 @@
+name: Sync wiki
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'wiki/**'
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: Checkout wiki
+        uses: actions/checkout@v3
+        with:
+          repository: "${{ github.repository }}.wiki"
+          path: "lichess-bot.wiki"
+      - name: Sync files to wiki
+        run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "Wiki sync"
+          cp -r $GITHUB_WORKSPACE/README.md $GITHUB_WORKSPACE/wiki/Home.md
```

Comment:
Instead of `cp -r $GITHUB_WORKSPACE/README.md $GITHUB_WORKSPACE/wiki/Home.md` over here I tried [this](https://github.com/TheYoBots/lichess-bot/commit/c9e4c9db281b966e2e282bdc81dcad9b1e0d26cf). But for some reason this works locally but on github I get the following error: `fatal: bad revision 'HEAD~1'`. Although that would be a much better change.